containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,18 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    doc_name="Apache-Daffodil-Extension-for-Visual-Studio-Code-$VERSION"
```

Comment:
The convention we've been using in this script is ALL_CAP variable names.

Reply:
I will fix that now, my apologies